be/src/column/array_column.cpp

```diff
@@ -427,6 +439,10 @@ Datum ArrayColumn::get(size_t idx) const {
     return Datum(res);
 }
 
+size_t ArrayColumn::get_element_size(size_t idx) const {
```

Comment:
better to add a DCHECK to check if idx is out of bounds

Reply:
done